pontoon/settings/dev.py

```diff
@@ -9,11 +9,14 @@
 
     # Adds various commands like e.g. shell which has all models loaded by default.
     'django_extensions',
+
+    # sslserver helps to develop features that require HTTPS.
+    'sslserver',
```

Comment:
It's usefull when you want to test if site behaves correctly under the ssl. I mostly required this to mimic mozilla-pontoon-staging.herokuapp.com on my localhost. It's only a helper -> It can be removed.